content/2019/05/24/generate.rst

```diff
@@ -0,0 +1,187 @@
+--------------
+
+Rapid Prototyping with Gatsby
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adekunle Oduye gave us an overview
+of rapid prototyping with Gatsby —
+create a design system,
+set up Gatsby,
+and start prototyping with code.
+It’s a super efficient way
+to communicate designs to developers.
+
+Aaron Irizarry
+--------------
+
+Groundwork: Building Foundations for High-Performing Teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aaron Irizarry confirmed my anecdotal experience
+about the factors that make a high performance team:
+create a psychologically safe work environment,
+avoid micromanagement and low standards,
+allow autonomy,
+provide clear expectations, stakes, goals, and evaluations,
+and teach people how to think.
+Thanks, OddBird,
+for challenging me to do my best work
+in a thoroughly supportive space. 🎉
+
+Jen Simmons
+-----------
+
+Everything You Know About Web Design Just Changed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Miriam and I had the privilege
+of watching Jen Simmons live-code
+a fiendishly difficult layout
+(that we selected for her)
+using CSS Grid.
+Wow!
+Such exciting new possibilities!
+
+Irene Pereyra
+-------------
+
+Getting Personal Projects Made
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The talk that stuck with me the most
+was Irene Pereyra’s Getting Personal Projects Made.
+OddBird has always tried to construct itself
+around team member needs
+instead of asking people
+to fit into a more standard structure —
+flexing as needs change,
+and staying committed
+when external forces push us to conform.
+We offer OSS project time
+to team members
+and are slowly building OddBooks,
+a collaborative writing tool.
+Her talk was like a shot of raw inspiration.
+
+Irene runs her own design studio
+with Anton Repponen.
+They made a commitment early on
+to focus 60% of their time on client work
+and 40% on personal projects
+and have stuck to it through the bombs and wild successes.
+For personal projects,
+they play around without clear goals.
+They pursue ideas that make them nervous.
+And they get help from friends and colleagues
+in adjacent fields.
+When Irene started on One Shared House,
+an interactive documentary about her childhood,
+she didn’t have high expectations.
+She certainly didn’t expect it
+to lead to a project with IKEA:
+One Shared House 2030,
+an imagining of co-living in the future.
+Some Anton & Irene projects are still costing them money,
+but satisfaction with their work is high.
+And what’s all this work for anyway,
+if not increased happiness?
+
+What's Next?
+------------
+
+Riding this wave of excitement,
+I plan to cash in on some of that OSS time OddBird offers
+to work on one of the projects
+that’s been percolating in my mind
+for a while.
+I’m thinking either an icon set
+(what new ideas can I bring to this crowded space?)
+or a tool for outlining writing projects
+(this one scares me the most!).
+Whatever happens,
+you’ll be among the first to know.
```

Comment:
This sign-off seems a bit forced, or overly personal.

Reply:
@jgerigmeyer Thanks, Jonny! Links added. Whenever you have a moment, I love another review, mostly to copy-edit my changes.